src/components/FooterModule.vue

```diff
@@ -25,19 +25,20 @@
         </ul>
       </div>
       <div class="column">
-        <p>Follow CVE on
-           <a href="/"><font-awesome-icon :icon="['fab', 'twitter']" /></a>
-           <a href="/"><font-awesome-icon :icon="['fab', 'github']" /></a>
-           <a href="/"><font-awesome-icon :icon="['fab', 'linkedin']" /></a>
-           <a href="/"><font-awesome-icon :icon="['fab', 'youtube']" /></a>
-        </p>
+        <h5 class="title is-5">Follow CVE on</h5>
+          <a class="add-space" href="/"><font-awesome-icon :icon="['fab', 'twitter']" /></a>
+          <a class="add-space" href="/"><font-awesome-icon :icon="['fab', 'github']" /></a>
+          <a class="add-space" href="/"><font-awesome-icon :icon="['fab', 'linkedin']" /></a>
+          <a class="add-space" href="/"><font-awesome-icon :icon="['fab', 'youtube']" /></a>
```

Comment:
Can you please make the class name descriptive of the element, such as: `class=social-media-icon`?